eiffel-vocabulary/EiffelIssueVerifiedEvent.md

```diff
@@ -18,43 +18,29 @@
 # EiffelIssueVerifiedEvent (IV)
 The EiffelIssueVerifiedEvent declares that an issue, typically a requirement, has been verified by some means. It is different from [EiffelTestCaseFinishedEvent](./EiffelTestCaseFinishedEvent.md) in that multiple test case executions may serve as the basis for a single verification or, conversely, multiple issues may be verified based on a single test case execution.
 
-## Data Members
-### data.issues
-__Type:__ Object[]  
-__Required:__ Yes  
-__Description:__ A list of verified (successfully or not) issues.
-
-#### data.issues.type
-__Type:__ String  
-__Required:__ Yes  
-__Legal values:__ BUG, IMPROVEMENT, FEATURE, WORK_ITEM, REQUIREMENT, OTHER  
-__Description:__ The type of issue.
+EiffelIssueVerifiedEvent has no data members, instead relying on its required link types. While "SUCCESSFUL_ISSUE", "FAILED_ISSUE", and "INCONCLUSIVE_ISSUE" are all marked as not required, at least one link of at least one of these types __must__ be present in an EiffelIssueVerifiedEvent.
```

Comment:
We try to stick to the SHALL, MUST, SHALL NOT, MAY etc. convention of IETF RFCs, rather than __shall__, __must__ etc.

Reply:
Changed __must__ -> MUST